pkg/security/rules/policy_monitor.go

```diff
@@ -244,3 +251,27 @@ func NewRuleSetLoadedEvent(ruleSets map[string]*rules.RuleSet, err *multierror.E
 	return events.NewCustomRule(events.RulesetLoadedRuleID, events.RulesetLoadedRuleDesc),
 		events.NewCustomEvent(model.CustomRulesetLoadedEventType, evt)
 }
+
+// NewHeartbeatEvents returns the rule (e.g. heartbeat) and a populated custom event for a heartbeat event
+func NewHeartbeatEvents(policies map[string]Policy) (*rules.Rule, []*events.CustomEvent) {
+
+	var evts []*events.CustomEvent
+
+	for _, policy := range policies {
+		var policyState = PolicyState{
+			Name:    policy.Name,
+			Version: policy.Version,
+			Source:  policy.Source,
+			Rules:   nil, // The rules that have been loaded at startup are not reported in the heartbeat event
```

Comment:
Very open to discussion but I'm thinking it's not super useful to include all the rules in every heartbeat event knowing it's already sent in ruleset_loaded